src/amo/pages/UserProfileEdit/index.js

```diff
@@ -612,7 +614,10 @@ export class UserProfileEditBase extends React.Component<InternalProps, State> {
                         // eslint-disable-next-line react/no-danger
                         dangerouslySetInnerHTML={sanitizeHTML(
                           i18n.sprintf(
-                            i18n.gettext(`You can change your email address on
+                            isMzaBranding()
+                              ? i18n.gettext(`You can change your email address on
+                          Mozilla Accounts. %(startLink)sNeed help?%(endLink)s`)
```

Comment:
Just caught something extra - it's "Mozilla **a**ccounts" (v.s. "Firefox **A**ccounts") - the case is important.  Please change everywhere it's written.